web/client/plugins/Widgets.jsx

```diff
@@ -160,6 +251,7 @@ class Widgets extends React.Component {
  * @memberof plugins
  * @name Widgets
  * @class
+ * @prop {boolean} [showMobileNavigation] show arrows to toggle between available widgets in mobile view. True by default.
```

Comment:
Why do we need an option to disable/enable only the arrows?

Reply:
It doesn't make sense to allow to disable select list completely, navigation between widgets is not possible without it. While arrows is an additional navigation method (to be honest I prefer it more than a select list) that could be disabled to free up some space for select list. Maybe we should allow to toggle between these two navigation methods or use both of them?

Reply:
I would prefer to remove this option for the moment as mentioned in the next review